Components/Avatar_and_badges/avatar_badges.css

```diff
@@ -0,0 +1,143 @@
+.content-bar {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  bottom: 500px;
+  left: 443px;
+  max-width: 800px;
```

Comment:
You can use layout utility style and use relative units instead of absolute units.